app/controllers/contents_controller.rb

```diff
@@ -0,0 +1,34 @@
+class ContentsController < ApplicationController
+  def index
+    @contents = Form.find(params[:id]).contents.includes(:submissions)
+  end
+
+  def create
+    @content = Form.find(params[:id]).contents.create(content_params)
+  end
+
+  def update
+    @content = Content.find(params[:id])
+    if @content.nil?
+      render json: {message: 'Cannot find content'}, status: :not_found
+    else
+      @content.update(content_params)
```

Comment:
Technically, I would even do a `if @content.update(content_params)` to make sure the program can handle the case when the update didn't work. Same thing for delete